tests/unit/explore_test.sh

```diff
@@ -67,9 +67,18 @@ function test_explore_files_under_git_repo()
   assertEquals "($LINENO)" "$MSG_OUT" "$output"
 
   output=$(explore_main 'GNU grep' '.' 'TEST_MODE')
-  expected_result="git grep -e 'GNU grep' -nI ."
+  expected_result="git grep --context 0 -e 'GNU grep' --line-number -I ."
   assertEquals "($LINENO)" "$expected_result" "$output"
 
+  # Test for non zero context value
```

Comment:
btw, the explore test has some bad test patterns, such as adding many checks in the same function. Since you are adding a new feature, I recommend you create specific functions for your tests. For example, you could create a function named `test_explore_with_git_and_context`. Ideally, we want one test function per use case since this keeps things easy to debug and update.